Model01-Firmware.ino

```diff
@@ -279,6 +284,27 @@ void hostPowerManagementEventHandler(kaleidoscope::HostPowerManagement::Event ev
   toggleLedsOnSuspendResume(event);
 }
 
+/** This 'enum' is a list of all the magic combos used by the Model 01's
+ * firmware The names aren't particularly important. What is important is that
+ * each is unique.
+ *
+ * These are the names of your magic combos. They will be used by the
+ * `USE_MAGIC_COMBOS` call below.
+ */
+enum { COMBO_PROTOCOL_TOGGLE };
+
```

Comment:
I'd add one line of doc here describing that it's to toggle between boot mode and NKRO mode.